### ISSUE TITLE
Clarifying the Search procedure

### ISSUE THREAD

**Issue by Guy-Dudeman** [mormaer/Mlem#327]
## Introduction
There have been a number of complaints about users not knowing where to click to search. 

## Requirements
The GUI needs to be more intuitive and help guide users to this function.

I propose adding a magnifying glass to the top row, which opens the search, and redesigning the search field within to look more like a search field, with a blinking cursor, so it is more obvious that that is where you type to search.
![LemmyHeader](https://github.com/mormaer/Mlem/assets/136025639/7bea8a3a-ccdf-4b44-be90-ff766a240455)
![LemmySearchFeeds](https://github.com/mormaer/Mlem/assets/136025639/f5b8d073-18f6-4581-acae-ac731620d1a9)

**Comment by Sjmarf** [comment]
I agree that the search should be more intuitive. Perhaps we could have a search bar at the bottom, like the one in the Maps app? The bottom of the phone is easier to reach, especially on Pro Max models.

It could take the place of the post title input that we currently have there - 99% of users will never make a post anyway.
